packages/opentelemetry-instrumentation-http/test/functionals/http-enable.test.ts

```diff
@@ -479,12 +479,29 @@ describe('HttpInstrumentation', () => {
         });
       }
 
-      it('should have 1 ended span when request throw on bad "options" object', () => {
+      it('should have 1 ended span when request throw on bad "options" object', done => {
         try {
-          http.request({ protocol: 'telnet' });
+          http.request({ headers: { cookie: undefined} }); // <===== this makes http.request throw
+          done('exception should have been thrown but did not');
```

Comment:
You could use [assert.throws](https://nodejs.org/dist/latest-v14.x/docs/api/assert.html#assert_assert_throws_fn_error_message) here.

Reply:
Indeed :) changed to `assert.throws` which is more appropriate here, thanks